portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +36,30 @@ function addRandomFact() {
   funFactContainer.innerText = funFact;
   funFactContainer.style.backgroundColor = "whitesmoke";
```

Comment:
Let's use single quote instead of double quote for string literals across this file.

Reference:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Done.